### PR TITLE
Update About Us

### LINKED ISSUE #21: Update About Us

### PR DESCRIPTION
**Pull request by kimberlytmq**
Added About Us section for Kimberly

Closes #21

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,53 +9,14 @@ We are a team based in the [School of Computing, National University of Singapor
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-## Project team
 
-### John Doe
+### Kimberly 
 
 <img src="images/johndoe.png" width="200px">
```
Comment (jtooya) [inline docs/AboutUs.md]: Could we change the src of the image to your .png file?

**Review by jtooya — CHANGES_REQUESTED** [review]
Just a few nits

**Review by jtooya — DISMISSED** [review]
LGTM

**Review by SlothyCat — APPROVED** [review]
LGTM

**Pull request merged**